data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/ParseTreeWalkerFactory.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.antlr.v4.runtime.tree.ParseTreeWalker;
+import org.springframework.context.annotation.Bean;
+
+import javax.inject.Named;
+
+@Named
+class ParseTreeWalkerFactory {
```

Comment:
Following Spring naming conventions, a class that creates and configures Beans usually has the suffix `Config` or `Configuration`. In our project I have been using `Configuration`